pages/common/tar.md

```diff
@@ -24,7 +24,7 @@
 
 `tar xf {{path/to/source.tar[.gz|.bz2|.xz]}} --directory={{path/to/directory}}`
 
-- [c]reate a compressed archive and write it to a [f]ile, using [a]rchive suffix to determine the compression program:
+- [c]reate a compressed archive and write it to a [f]ile, using file suffix to [a]utomatically determine the compression program:
```

Comment:
```suggestion
- [c]reate a compressed archive and write it to a [f]ile, using filename's suffix to [a]utomatically determine the compression program:
```

Reply:
I kinda forgot this PR 😅 but I think "file extension" is better and more commonly used

Reply:
You're totally right!